app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class CartsController < ApplicationController
```

Comment:
Let's make it CartController, as we are working with one cart, and don't create new one's